packages/fxa-payments-server/src/components/SubscriptionForm/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { injectStripe, CardElement } from 'react-stripe-elements';
+
+class SubscriptionForm extends React.Component {
+  
+  render() {
+    const {
+      plansByProductId,
+    } = this.props;
+    
+    return (
+      <form onSubmit={this.handleSubmit}>
```

Comment:
I usually name the method `onSubmit` just to reduce the number of names.

Is `.bind(this)` no longer necessary? I feel like I read a headline but never read the article...

Reply:
Yeah, this is literally just copypasta from the [react-stripe-elements example code](https://github.com/stripe/react-stripe-elements#setting-up-your-payment-form-injectstripe) - no reason to leave it like this.